composer.json

```diff
@@ -12,12 +12,14 @@
     ],
     "scripts": {
         "test": "phpunit",
-        "phpstan": "phpstan analyse --level=7 src"
+        "phpstan": "phpstan analyse --level=7 src",
```

Comment:
can you draft a quick config? that way we can have a baseline easily and use larastan too

Reply:
Ah, there is a config already `phpstan.neon.dist`. Can you import larastan there?

Reply:
28c1fd4ed6ef9d5d9ef71adcc66b8c03b30b20ab

I had to remove some Laravel-related parameters

<img width="784" alt="Screenshot 2022-08-03 at 14 18 05" src="https://user-images.githubusercontent.com/14861869/182605736-1c59a5d5-6e51-45a9-aaba-bdd682ed56b6.png">